examples/commander.yaml

```diff
@@ -24,4 +24,9 @@ tests:
   it should skip:
     command: echo "I should be skipped"
     stdout: I should be skipped
-    skip: true
\ No newline at end of file
+    skip: true
```

Comment:
Is this example still intended to work?
I am not sure the relative path to the fixtures would work

Reply:
So the reason why this is needed is because we're not properly mocking ioutil.Readfile in the unit tests for `app.TestCommand`... 100% my fault. [Here is the exact line](https://github.com/commander-cli/commander/blob/master/pkg/app/test_command_test.go#L64). I opened #149 for this already and pasted the failing build that showed why this pathing was need.

Reply:
ahhh okay, than it can be merged 👍